### ISSUE TITLE
Test install wmic and wmiplus plugins on Ubuntu 16.04+

### ISSUE THREAD

**Issue by pablodav** [CoffeeITWorks/ansible_nagios4_server_plugins#3]
The current scenario for molecule doesn't activates the install of this plugin:

scenario:
https://github.com/CoffeeITWorks/ansible_nagios4_server_plugins/blob/master/molecule/default/playbook.yml

It should have the var:

```yaml
nagios_plugins_install_checkwmiplus: True
```

What we will not test:

```yaml
nagios_plugins_USER8_wmi_username: 'someuser' # used by commands for checkwmiplus
nagios_plugins_USER9_wmi_password: "{{ VAULT_nagios_plugins_USER9_wmi_password }}" # used by commands for checkwmiplus
```

**Comment by pablodav** [comment]
Apparent fixes:

https://medium.com/@alberto.schiabel/how-to-compile-wmic-on-a-modern-gnu-linux-system-d4b0783bbf64

New versions:

https://github.com/astbss/wmic

**Comment by pablodav** [comment]
Other useful information:

https://techedemic.com/2012/11/05/installing-wmic-in-ubuntu-12-04-lts-64-bit-desktop/

**Comment by pablodav** [comment]
https://www.si458.co.uk/?p=134

https://www.shellandco.net/blog/2017/01/19/wmic-command-ubuntu-16-04-lts/

**Comment by pablodav** [comment]
resolved in version 1.1.4
